dotnet/src/webdriver/CookieJar.cs

```diff
@@ -61,9 +69,9 @@ public void AddCookie(Cookie cookie)
         /// Delete the cookie by passing in the name of the cookie
         /// </summary>
         /// <param name="name">The name of the cookie that is in the browser</param>
-        public void DeleteCookieNamed(string name)
+        public void DeleteCookieNamed(string? name)
         {
-            Dictionary<string, object> parameters = new Dictionary<string, object>();
+            Dictionary<string, object?> parameters = new Dictionary<string, object?>();
```

Comment:
I wasn't sure how to handle this. When `name` is `null`, nothing seems to happen. So maybe we can do the same as `DeleteCookie(Cookie)` below?

Reply:
No more relevant to tthis PR.